### PR TITLE
PHPUnit tests for context

### PR DESCRIPTION
**Pull request by loic425**
| Q               | A
| --------------- | -----
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

Based on #815

### CODE REVIEW

File: src/Component/tests/Context/Option/MetadataOptionTest.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<?php
```
Comment (diimpp) [inline src/Component/tests/Context/Option/MetadataOptionTest.php:1]: I think it needs to be removed as well.

`src/Component/tests/spec/Context/Option/MetadataOptionSpec.php`

Comment (loic425) [reply src/Component/tests/Context/Option/MetadataOptionTest.php:1]: You're right, normally, I move the Phpspec testing class and adapt it to PHPUnit, but I did it wrong with this one.

Comment (loic425) [reply src/Component/tests/Context/Option/MetadataOptionTest.php:1]: done

**Review by diimpp — APPROVED** [review]

**Comment by lchrusciel** [comment]
Thank you, @loic425!

**Pull request merged**